RELEASE_NOTES

```diff
@@ -3,8 +3,64 @@ For more detailed information, please see the git log.
+    awscli (#3645), behave (#3751), Blosc (#3785), bokeh (#3790), Cantera (#3655),
+    Cargo (#3764), dadi v1.7.0, distributed (#3786), ea-utils (#3634), Elk (#3644),
+    FGSL (#3638), gencore_variant_detection (#3337), help2man (#3768), lbzip2 (#3791),
+    Log-Log4perl (#3574), Minimac2 (#3783), mypy (#3694), OBITools (#3573),
+    perl-app-cpanminus (#3337), PGDSpider (#3625), prokka (#3755), Reads2snp (#3609),
+    spglib-python (#3620), SUNDIALS (#3654, #3655), SelEstim (#3626), XMLStarlet (#3797),
+    x265 (#3090)
+- added easyconfigs for new 'intelcuda' toolchain (#3750)
+- added new easyconfigs for existing toolchains:
+  - goolfc/2016.08 (#3796), goolfc/2016.10 (#3666, #3775), intel/2017.00 (#3543), intel/2017.01 (#3757),
+    iomkl/2016.09-GCC-4.9.3-2.25 (#3680), iomkl/2016.09-GCC-5.4.0-2.26 (#3772)
+- added additional easyconfigs for various supported software packages, including:
+  - Advisor 2017 update 1, Amber 16, ATLAS 3.10.2, GROMACS 2016, Octave 4.0.3,
+    OpenFOAM 3.0.1, PyTables 3.3.0, QuantumESPRESSO 6.0, Rust v1.12.1
+- various other enhancements, including:
+  - STREAM builds using ~56GiB and ~111GiB (#3670)
+- various bug fixes, including:
+  - fix source spec in VASP easyconfig, ensure static linking with Intel MKL (#3381)
+  - fix source URL in GCCcore 6.2.0 easyconfig (#3608)
+  - correct STAMP dependency in i-cisTarget, must be 1.3 (#3613)
+  - consistently specify to use -fgnu89-inline flag in M4 1.4.17 easyconfigs (#3623)
+  - fix source URLs for Cython (#3636)
+  - add Bison as build dep and M4 as runtime dep for flex 2.6.0 (#3656)
+  - enable parallel building of flex 2.6.0 (#3630)
+  - add zlib and bzip2 dependencies to X11 bundle (#3662)
+  - use 'letter_dir_for' function rather than just grabbing 1st letter of software name in easyconfigs tests (#3664)
+  - add patch to fix typo in GRIT 2.0.5 (#3675)
+  - fix typo in patch for WRF 3.8.0 (#3702)
+  - use $CC,$CXX rather than $I_MPI_CC,$I_MPI_CXX in patch for OpenFOAM 4.0 (#3703)
+  - patch FLTK to fix 'undefined symbol' issue when building Octave (#3704)
+  - include Pillow as a proper dep for scikit-image rather than as extension, since it has deps itself (#3723)
+  - update Travis config w.r.t. changes framework config defaults and required Lmod version (#3773)
+  - don't limit parallellism to 4 in recent GCC easyconfigs (#3776, #3777, #3778)
+  - include M4 as dependency in flex 2.5.39 easyconfigs + fix consistency issues (#3782)
+  - consistently apply patch for ncurses 6.0 (#3792)
+  - eliminate dependency on mpi-mic-rt in ifort (#3793)
+  - include Autotools as build dependency in all beagle-lib and MrBayes easyconfigs (#3794)
+  - make OpenBLAS use the LAPACK version specified in the easyconfig (v0.2.18 & v0.2.19) (#3795)
+  - include original download URL for ISL source tarball in GCC easyconfigs (#3798)
+  - disable installing libiberty for binutils built with intel toolchain (#3802)
```

Comment:
@fgeorgatos that'll have to wait until v3.0.1 or v3.1, whichever happens first, sorry

People can use `--from-pr` on it though (if they know about it ;))